django_saml2_auth/views.py

```diff
@@ -183,6 +186,16 @@ def acs(r):
     else:
         return HttpResponseRedirect(get_reverse([denied, 'denied', 'django_saml2_auth:denied']))
 
+    if settings.SAML2_AUTH.get('USE_JWT') is True:
+        # We use JWT auth send token to frontend
+        jwt_token = jwt_encode(target_user)
+        query = '?uid={}&token={}'.format(target_user.id, jwt_token)
+
+        frontend_url = settings.SAML2_AUTH.get(
+            'FRONTEND_URL', next_url)
+
+        return HttpResponseRedirect(frontend_url+query)
```

Comment:
How does this work then, since now your skipping the whole SAML response. If the client initializes a SAML flow wouldn't it expect it to finish too?